src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Export/ExportJobTaskTests.cs

```diff
@@ -1454,6 +1476,243 @@ public async Task GivenAGroupExportJobWithANonExistantGroup_WhenExecuted_ThenThe
             Assert.Equal("test", _exportJobRecord.FailureDetails.FailureReason);
         }
 
+        [Fact]
+        public async Task GivenAnonymizedExportJob_WhenExecuted_ThenItShouldExportAllAnonymizedResources()
+        {
+            bool capturedSearch = false;
+
+            var exportJobRecordWithOneResource = new ExportJobRecord(
```

Comment:
There is a helper method (CreateExportJobRecord) that can save you some lines of code here and in the other new tests.

Reply:
Thanks, updated.